client/src/components/feddback-comment.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { Container } from 'react-bootstrap'
+import '../style/feedback.css'
+
+class FeedbackComment extends Component {
+
+
+    render() {
+
+        return (
+
+            <Container className="feedbackUserContainer">
+                <img src="/img/defAvatar.png" width="50px" height="50px" alt="defAvava"></img>
+                <div className="feedbackMess">
+                    <strong className="nameDate">{this.props.item.userName}</strong>
+                    <small className="nameDate">{this.props.item.date}</small>
+                    <hr className="hrStyle" />
+                    <div>{this.props.item.description}</div>
+                    <hr className="hrStyle" />
```

Comment:
Sorry, but I don't know the purpose of code in lines 17 and 19. Give me a hint, please.

Reply:
http://htmlbook.ru/html/hr

Reply:
Thanks!